<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsMesh3dPropsWidget</class>
 <widget class="QWidget" name="QgsMesh3dPropsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mGroupBoxTrianglesSettings">
     <property name="title">
      <string>Triangles Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="mGroupBoxWireframe">
        <property name="title">
         <string>Wireframe Line Width And Color</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QgsDoubleSpinBox" name="mSpinBoxWireframeLineWidth" native="true">
           <property name="suffix" stdset="0">
            <string> px</string>
           </property>
           <property name="value" stdset="0">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsColorButton" name="mColorButtonWireframe"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="mChkWireframe">
        <property name="text">
         <string>Wireframe</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="mChkSmoothTriangles">
        <property name="text">
         <string>Smooth triangles</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mGroupBoxVerticaleSettings">
     <property name="title">
      <string>Vertical Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Vertical Scale</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QgsDoubleSpinBox" name="mSpinBoxVerticaleScale" native="true">
        <property name="maximum" stdset="0">
         <double>9999999.990000000223517</double>
        </property>
        <property name="value" stdset="0">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dataset Group For Vertical Value</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="mComboBoxDatasetVertical"/>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="mCheckBoxVerticalMagnitudeRelative">
        <property name="text">
         <string>Dataset Value Relative To Vertices Z Value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="mGroupBoxTextureSettings">
     <property name="title">
      <string>Rendering Color Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Rendering Style</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mComboBoxTextureType"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="mGroupBoxColorRampShader">
        <property name="title">
         <string>Color Ramp Shader</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>9</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item row="1" column="0" colspan="5">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mColorRampShaderMinEdit"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Min</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Max</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="mColorRampShaderMinMaxReloadButton">
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../images/images.qrc">
             <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="5">
          <widget class="QgsColorRampShaderWidget" name="mColorRampShaderWidget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="mColorRampShaderMaxEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QWidget" name="mMeshSingleColorWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mesh Color</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsColorButton" name="mMeshSingleColorButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorRampShaderWidget</class>
   <extends>QWidget</extends>
   <header>raster/qgscolorrampshaderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
